src/main/java/gift/Service/CategoryService.java

```diff
@@ -0,0 +1,56 @@
+package gift.Service;
+
+import gift.ConverterToDto;
+import gift.DTO.Category;
+import gift.DTO.CategoryDto;
+import gift.Repository.CategoryRepository;
+import java.util.List;
+import org.springframework.dao.EmptyResultDataAccessException;
+import org.springframework.stereotype.Service;
+
+@Service
+public class CategoryService {
+
+  private final CategoryRepository categoryRepository;
+
+  public CategoryService(CategoryRepository categoryRepository) {
+    this.categoryRepository = categoryRepository;
+  }
+
+  public List<CategoryDto> getAllCategories() {
+    List<Category> categories = categoryRepository.findAll();
+    List<CategoryDto> categoryDtos = categories.stream().map(ConverterToDto::convertToCategoryDto)
```

Comment:
```suggestion
    List<CategoryDto> categoryDtos = categories.stream()
    .map(ConverterToDto::convertToCategoryDto)
   .toList()
```
이런식으로 개행을 한줄의 점 하나씩으로 해서 하면 더 가독성이 높아집니다~

Reply:
오..감사합니다! 앞으로 해당사항 반영하며 코드를 구현하겠습니다